### PR TITLE
Blog post:  Rune frequency

### PR DESCRIPTION
**Pull request by sigilante**

### CODE REVIEW

File: content/blog/rune-frequency-202212.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+(Bear with me on that graph:  it's rather hard to display order runes in any sort of coherent way I've come up with so far.)
+
+
+
+##  Analysis
+
+There aren't any real surprises here.  The most frequent runes reflect the most common design patterns:
+
+- `%` cen rune calls tend to route through [`%:` cencol](https://developers.urbit.org/reference/hoon/rune/cen#-cencol) since the irregular form `(fun 1 2)` desugars to `%:`.
+- [`:*` coltar](https://developers.urbit.org/reference/hoon/rune/col#-coltar) serves similarly as the desugaring of lists constructed by `~[1 2 3]`.
```
Comment (tinnus-napbus) [inline content/blog/rune-frequency-202212.md:185]: ordinary cell-forming like `[1 2 3]` is the irregular form of `:*`, it's more likely from that than lists

File: content/blog/rune-frequency-202212.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+- [`%=` centis](https://developers.urbit.org/reference/hoon/rune/cen#-centis) is invoked through the irregular `$()` expression resets.  These are most commonly employed in gates and traps as a recursion.
```
Comment (tinnus-napbus) [inline content/blog/rune-frequency-202212.md:186]: this is used very extensively for modifying legs as well as recursion, maybe good to mention both

**Review by tinnus-napbus — CHANGES_REQUESTED** [review]
LGTM & is interesting. Just a couple of minor comments

File: content/blog/rune-frequency-202212.md (hunk 2 above)
Comment (tinnus-napbus) [reply content/blog/rune-frequency-202212.md:186]: eg `this(thing new-thing)` in agents and everywhere with engine patterns

File: content/blog/rune-frequency-202212.md (hunk 1 above)
Comment (sigilante) [reply content/blog/rune-frequency-202212.md:185]: yeah, I was def. thinking of `:~` when I wrote that

**Comment by sigilante** [comment]
Thanks for comments.  I'll release on Jan 12.

**Pull request merged**